src/domain/cohort.js

```diff
@@ -4,12 +4,20 @@ import dbClient from '../utils/dbClient.js'
  * Create a new Cohort in the database
  * @returns {Cohort}
  */
-export async function createCohort() {
-  const createdCohort = await dbClient.cohort.create({
-    data: {}
-  })
+export async function createCohort(cohortName) {
+  cohortName = String(cohortName)
 
-  return new Cohort(createdCohort.id)
+  let createdCohort
+
+  try {
+    createdCohort = await dbClient.cohort.create({
+      data: { cohortName: cohortName || '' }
```

Comment:
Let's also remove the default value for a cohort name - throw an error if it's anything but a string of 1 or more chars.

Reply:
Error thrown if String has less than 1 characters.